model/pixtral/imageproc.go

```diff
@@ -0,0 +1,68 @@
+package pixtral
+
+import (
+	"fmt"
+	"image"
+	_ "image/jpeg"
+	_ "image/png"
+	"io"
+	"math"
+
+	"github.com/ollama/ollama/model/imageproc"
+)
+
+func getNumImageTokens(imageSize, patchSize image.Point) image.Point {
```

Comment:
prefixing `get` may be extraneous here

```suggestion
func numImageTokens(imageSize, patchSize image.Point) image.Point {
```